src/tableHeaderCells.js

```diff
@@ -7,7 +7,7 @@ const TableSortHeaderCell = ({columnNumber, headerText, width, orderedColumnInde
     className={`header${columnNumber}`}
     flexBasis={width} flexShrink={100} flexGrow={100}
     onClick={() => onClick(columnNumber)}>
-    <div style={{display: `ruby`}}>
+    <div>
```

Comment:
I added the `ruby` style to make sure the header text is displayed in one line. Are you sure you would like to delete it?

Reply:
[Yes. :smile:](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-ruby-element)